texture_manager.c

```diff
@@ -750,7 +752,7 @@ void texture_manager_tick(texture_manager *manager) {
 
         // zero the epoch used bins
         memset(m_epoch_used, 0, sizeof(m_epoch_used));
-    } else if (highest > manager->max_texture_bytes) {
+    } else if (highest > manager->max_texture_bytes || overLimit) {
```

Comment:
There is an issue in this change.

Trying to get more more memory when device has less memory available will kill the app.